src/rpc/server.cpp

```diff
@@ -496,8 +496,6 @@ UniValue CRPCTable::execute(const std::string &strMethod, const UniValue &params
     {
         throw JSONRPCError(RPC_MISC_ERROR, e.what());
     }
-
-    g_rpcSignals.PostCommand(*pcmd);
```

Comment:
Wasn't this a bug? `PostCommand` and `OnPostCommand` should be deleted if they are never called.

Reply:
Yeah, I think you’re right – I rewrote this to reach the deleted code instead of deleting it.

Reply:
Please backport https://github.com/bitcoin/bitcoin/pull/9575 instead.

Reply:
See #6673.